dis_snek/models/discord_objects/components.py

```diff
@@ -382,6 +384,22 @@ def get_components_ids(component: Union[str, dict, list, InteractiveComponent])
         raise ValueError(f"Unknown component type of {component} ({type(component)}). " f"Expected str, dict or list")
 
 
+def get_messages_ids(message: Union[int, "Message", list]) -> Iterator[int]:
```

Comment:
I feel like we can just use the generic `to_snowflake_list()`?

Reply:
How do I do that?

Reply:
So instead of
```py
message_ids = list(get_messages_ids(messages)) if messages else None
```
I should do
```py
messages_ids = to_snowflake_list(messages) if messages else None
```
? Or is this wrong?

Reply:
Done this, time to test it

Reply:
Oops, I forgot to reply, works